src/main/java/com/networknt/schema/ExclusiveMaximumValidator.java

```diff
@@ -35,13 +35,22 @@ public class ExclusiveMaximumValidator extends BaseJsonValidator {
 
     private final ThresholdMixin typedMaximum;
 
+    
+    private static boolean checkSchemaNodeLongOrInt(JsonNode schemaNode) {
+        return (schemaNode.isLong() || schemaNode.isInt()) ;
+    }
+    private boolean checkIntEqNodeFieldType() {
+        return JsonType.INTEGER.toString().equals(getNodeFieldType())  ;
+    }
+
+
     public ExclusiveMaximumValidator(SchemaLocation schemaLocation, JsonNodePath evaluationPath, final JsonNode schemaNode, JsonSchema parentSchema, ValidationContext validationContext) {
         super(schemaLocation, evaluationPath, schemaNode, parentSchema, ValidatorTypeCode.EXCLUSIVE_MAXIMUM, validationContext);
         if (!schemaNode.isNumber()) {
             throw new JsonSchemaException("exclusiveMaximum value is not a number");
         }
         final String maximumText = schemaNode.asText();
-        if ((schemaNode.isLong() || schemaNode.isInt()) && (JsonType.INTEGER.toString().equals(getNodeFieldType()))) {
+        if (checkSchemaNodeLongOrInt(schemaNode) && checkIntEqNodeFieldType()) {
```

Comment:
The changes from `if ((schemaNode.isLong() || schemaNode.isInt()) && (JsonType.INTEGER.toString().equals(getNodeFieldType()))) {` to `if (checkSchemaNodeLongOrInt(schemaNode) && checkIntEqNodeFieldType()) {` also seem rather arbitrary and I don't really see this adding any readability or promote reuse.